### PR TITLE
Update feature-requests.md

### PR DESCRIPTION
**Pull request by lfbrock**
Remove reference to Jira & replace with productboard

### CODE REVIEW

File: operations/research-and-development/product/product-planning/feature-requests.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ At Mattermost, we have two main places feature requests are tracked.
 
 The first one is [Uservoice](https://mattermost.uservoice.com/forums/306457-general). Uservoice is open to the public, and anyone is able to submit a request or upvote existing requests. We often send community members to this site when they request features. 
 
-The second place is in a "Customer Request" Jira project. This is an internal Jira project, so only Mattermost Staff can view it. Anytime a customer or potential customer has a request, it should be added to this project.
+The second place is in Productboard. This is an internal tool, so only Mattermost Staff can view it. Anytime a customer or potential customer has feedback, it should be added to Productboard (see [documentation](https://handbook.mattermost.com/operations/research-and-development/product/how-to-guides-for-product/how-to-use-productboard#2b-adding-feedback)).
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-planning/feature-requests.md]:
```suggestion
The second place is in ProductBoard. This is an internal tool, so only Mattermost staff can view it. Anytime a customer or potential customer has feedback, [it should be added to ProductBoard](https://handbook.mattermost.com/operations/research-and-development/product/how-to-guides-for-product/how-to-use-productboard#2b-adding-feedback). 
```

File: operations/research-and-development/product/product-planning/feature-requests.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ At Mattermost, we have two main places feature requests are tracked.
 
 The first one is [Uservoice](https://mattermost.uservoice.com/forums/306457-general). Uservoice is open to the public, and anyone is able to submit a request or upvote existing requests. We often send community members to this site when they request features. 
 
-The second place is in a "Customer Request" Jira project. This is an internal Jira project, so only Mattermost Staff can view it. Anytime a customer or potential customer has a request, it should be added to this project.
+The second place is in Productboard. This is an internal tool, so only Mattermost Staff can view it. Anytime a customer or potential customer has feedback, it should be added to Productboard (see [documentation](https://handbook.mattermost.com/operations/research-and-development/product/how-to-guides-for-product/how-to-use-productboard#2b-adding-feedback)). 
 
 The product management team uses the information from both Uservoice and Jira to help prioritize features for the roadmap.
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-planning/feature-requests.md]:
```suggestion
The Product Management team uses the information from both Uservoice and ProductBoard to help prioritize features for the roadmap. 
```

File: operations/research-and-development/product/product-planning/feature-requests.md [hunk 3]
```diff
@@ -43,38 +43,3 @@ If a deal is dependent on committing to a concrete timeline for a feature, the s
 The product management team will then evaluate the ask, and determine if we can commit to a timeline or if the feature does not fit on our roadmap.
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-planning/feature-requests.md]:
```suggestion
The Product Management team will then evaluate the ask, and determine if we can commit to a timeline or if the feature does not fit on our roadmap. 
```

**Comment by justinegeffen** [comment]
General comment:
I changed "Productboard" to "productboard" based on their spelling of it. It does mean that the name blends into the content and it may not be immediately clear that it's the name of the tool. 

I'm 4/5 for keeping with the way the tool is spelled but wondering if we need a formatting convention for tool names (e.g. bolding them). I'm 1/5 on that as it'd need to be applied across the board (e.g. to BambooHR, Jira, Uservoice etc). 

Formatting comment:
There are some formatting changes I'd like to make but that is a separate PR.

**Review by jasonblais — DISMISSED** [review]

**Comment by justinegeffen** [comment]
Update on this: I have changed productboard to ProductBoard based on the current usage in the [how to](https://handbook.mattermost.com/operations/research-and-development/product/how-to-guides-for-product/how-to-use-productboard#1h-key-links-in-productboard) doc. 

Discussion around the correct capitalization will be in a separate PR.

**Review by justinegeffen — APPROVED** [review]

**Review by jasonblais — APPROVED** [review]

**Pull request merged**